src/main/kotlin/me/ezra_home/retail_software_solution/model/entity/FruitEntity.kt

```diff
@@ -0,0 +1,51 @@
+package me.ezra_home.retail_software_solution.model.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.Size
+import org.hibernate.annotations.ColumnDefault
+import java.math.BigDecimal
+import java.time.OffsetDateTime
+import java.util.*
+
+@Entity(name = "FruitEntity")
```

Comment:
we don't really need an entity name. It will be something extra to track without getting a value out of it